public/css/footer.css

```diff
@@ -0,0 +1,105 @@
+/***************** Footer Style ****************/
+
+footer {
+    background-color: #575757;
+    width: 100%;
+    height: 150px;
+    display: flex;
+    /* align-content: center; */
+  }
+
+  footer .container {
+    width: 90%;
+    margin: 0 auto;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+  }
+
+  footer .container .logo-container {
+    width: fit-content;
+    margin-left: 150px;
+  }
+
+  footer .container .logo-container span {
+    display: block;
```

Comment:
What is the main idea of using a span and using disply block
the div could do the same without the need to use disply property!!